shared/src/types/admin/User.ts

```diff
@@ -10,30 +10,30 @@ export enum UserStatusKey {
 
 export type UserAddress = {
 	city?: string;
-	country?: string;
+	country?: string; // TODO: proper typing
```

Comment:
👍 or at least define in a comment that it needs to be two-letter iso code in uppercase.